### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by anshgoyalevil**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [![AsyncAPI Banner and Logo](./public/img/logos/github-repobanner-website.png)](https://www.asyncapi.com)
 
----
+
```
Comment (TRohit20) [inline README.md:3]: Removing the new line to remove the extra space between logo and socials. Seems redundant. Other than that. LGTM
```suggestion
```

**Review by TRohit20 — APPROVED** [review]
Commit the suggested change, post which, LGTM

File: README.md (hunk 1 above)
Comment (anshgoyalevil) [reply README.md:3]: 😆 Thanks for the review man

**Comment by TRohit20** [comment]
Ah sheesh, I was just going through all the PRs in notifications, my bad. @anshgoyalevil

**Pull request closed without merging**